src/shell.css

```diff
@@ -13,6 +13,8 @@ canvas.emscripten { border: 0px none; background-color: black; }
 #emscripten_logo {
   display: inline-block;
   margin: 0;
+  padding: 6px;
+  width: 265px;
```

Comment:
Does the sizer of the image not already determine the size here?  i.e. is this duplicating the information already encoded in the image?

Reply:
No. When I resized the image to 265 pixels width, the drop-shadow of the "powered by" text didn't come out very well. So I left the image larger and scale it down via CSS to get a better result. I didn't want to stray away too much from the original look.